contrib/storage-kafka/pom.xml

```diff
@@ -88,14 +88,7 @@
     <dependency>
       <groupId>org.apache.curator</groupId>
       <artifactId>curator-test</artifactId>
-      <version>${curator.test.version}</version>
       <scope>test</scope>
```

Comment:
It is already specified in dependencyManagement. No need to use it for every `curator-test` in every module anymore.